fe2-android/app/src/test/java/com/github/dedis/popstellar/utility/handler/ElectionHandlerTest.java

```diff
@@ -72,11 +73,12 @@ public void setup() throws GeneralSecurityException, IOException {
     lenient().when(keyManager.getMainKeyPair()).thenReturn(SENDER_KEY);
     lenient().when(keyManager.getMainPublicKey()).thenReturn(SENDER);
 
-    laoRepository = new LAORepository();
+    laoRepo = new LAORepository();
 
     when(messageSender.subscribe(any())).then(args -> Completable.complete());
 
-    laoRepository = new LAORepository();
+    messageRepo = new MessageRepository();
+    laoRepo = new LAORepository();
```

Comment:
isn't it duplicated from line 76 ? if it's intentional, please leave a comment.

Reply:
Removed 😄